packages/svelte/package.json

```diff
@@ -2,67 +2,97 @@
   "name": "@ai-sdk/svelte",
   "version": "1.1.19",
   "license": "Apache-2.0",
-  "sideEffects": false,
```

Comment:
Switched to a pattern to indicate CSS files produce side effects (this library currently does not output CSS, so there should be no net change here)